package-lock.json

```diff
@@ -95,6 +95,9 @@
         "jest-preview": "^0.3.1",
         "jquery": "^3.2.1",
         "postcss-modules": "^6.0.0"
+      },
+      "engines": {
+        "node": ">=20.x"
```

Comment:
Remove this line, you should not make any changes to `package-lock.json`

Reply:
Yes

Reply:
@rishav-jha-mech 
Removed the above lines from `package-lock.json`